rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -153,11 +166,113 @@ def _decode_name_renewed(
 
         return None, []
 
+    def _decode_ens_registry_with_fallback_event(
+            self,
+            tx_log: EthereumTxReceiptLog,
+            transaction: EvmTransaction,
+            decoded_events: List[HistoryBaseEntry],
+            all_logs: List[EthereumTxReceiptLog],  # pylint: disable=unused-argument
+            action_items: Optional[List[ActionItem]],  # pylint: disable=unused-argument
+    ) -> Tuple[Optional[HistoryBaseEntry], List[ActionItem]]:
+        if tx_log.topics[0] == NEW_RESOLVER:
+            node = tx_log.topics[1]
+            try:
+                ens_name = ENS_REVERSE_RESOLVER.call(
+                    manager=self.ethereum_manager,
+                    method_name='name',
+                    arguments=[node],
+                )
+            except RemoteError as e:
+                log.debug(f'Failed to decode ENS set-text event due to {str(e)}')
+                return None, []
+
+            if ens_name == '':
+                # By checking the contract code, I don't think it can happen. But just in case.
+                return None, []
+
+            # Not able to give more info to the user such as address that was set since
```

Comment:
Do you have a test for this? Can I see some events?

Reply:
Yeah, I did add a test. Transaction used in the test is https://etherscan.io/tx/0xae2cd848ce02c425bc50a8f46f8430eec32234475efb6fcff28315d2791329f6